src/test/kotlin/org/jetbrains/research/transformations/anonimization/AnonymizationTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.research.transformations.anonimization
+
+import PythonTransformationsTest
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.Util.getContentFromFile
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.io.File
+import java.util.stream.Stream
+
+
+internal class AnonymizationTest : PythonTransformationsTest() {
+
+    companion object {
+        @JvmStatic
+        fun getResourceFolder(): Stream<Arguments> {
+            return Stream.of(
+                Arguments.of(
+                    getResourcePath("in_1.py", ::AnonymizationTest),
+                    getResourcePath("out_1.py", ::AnonymizationTest)
+                )
+            )
+        }
+    }
+
+    @ParameterizedTest(name = "Apply anonymization test")
+    @MethodSource("getResourceFolder")
+    fun applyAnonymizationTests(inFile: File, outFile: File) {
+        transformCode(inFile, outFile, Anonymization::apply)
+    }
+
+    @ParameterizedTest(name = "Inverse apply anonymization test")
+    @MethodSource("getResourceFolder")
+    fun inverseApplyAnonymizationTests(outFile: File, inFile: File) {
+        transformCode(inFile, outFile) { ctx: TreeContext, _: Boolean ->
+            (Anonymization::apply)(ctx, true)
+            (Anonymization::inverseApply)(ctx)
+        }
+    }
```

Comment:
Are these methods common for all transformation, not only for the Anonymization? If so, shouldn't we move them to the base class?

Reply:
I am not sure about this. I was thinking about this and created the transformCode function. But I can move it, of course

Reply:
Let's keep it like this for now and think about it later when we add the other transformations